chapter_04/p09_bst_sequences.py

```diff
@@ -39,6 +39,34 @@ def weave(first, second, prefix, results):
     return results
 
 
+ret_backtracking = []  # global variable for backtracking
```

Comment:
if you nest the function definitions then you can keep this out of the global scope